### DISCUSSION TITLE
sortablejs in devDependency

### DISCUSSION THREAD

**Discussion by mihalcea-bogdan-daniel** [vueform/vueform#100] [Questions] [answered]
In the package.json file I noticed that sortablejs is inside "[devDependencies](https://github.com/mihalcea-bogdan-daniel/vueform/blob/30363b35334be187eb8c8a194fcf2be9a04c1894/package.json#L110)". Isn't this a core package that runs the form editor's drag and drop part?

**Comment by adamberecz** [comment, marked as answer]
No, that is only required for list sorting and bundled with the Vueform dist file (that's why it's only in the devDeps). The drag and drop builder is a different package and uses custom dnd: https://github.com/vueform/builder
